Intacct.SDK/Functions/AccountsReceivable/ArAdvanceCreate.cs

```diff
@@ -0,0 +1,72 @@
+﻿/*
+ * Copyright 2020 Sage Intacct, Inc.
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License"). You may not
+ * use this file except in compliance with the License. You may obtain a copy 
+ * of the License at
+ * 
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * or in the "LICENSE" file accompanying this file. This file is distributed on 
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either 
+ * express or implied. See the License for the specific language governing 
+ * permissions and limitations under the License.
+ */
+
+using Intacct.SDK.Xml;
+
+namespace Intacct.SDK.Functions.AccountsReceivable
+{
+    
+    public class ArAdvanceCreate : AbstractArAdvance
+    {
+        public ArAdvanceCreate(string controlId = null) : base(controlId)
+        {
+        }
+        
+        public override void WriteXml(ref IaXmlWriter xml)
+        {
+            xml.WriteStartElement("function");
+            xml.WriteAttribute("controlid", ControlId, true);
+
+            xml.WriteStartElement("create");
+            
+            xml.WriteStartElement("ARADVANCE");
```

Comment:
see if these literals you are using like `ARADVANCE` , `CUSTOMERID`, etc.. already have a constant dictionary that you can use. If there are no constants defined for these, don't worry about it

Reply:
There aren't any, unfortunately